.github/workflows/client-tests.yml

```diff
@@ -98,6 +98,10 @@ jobs:
       if: (matrix.os == 'ubuntu-latest')
       run: |
         # Jenkins can perform the full jujud testing.
-        go test -v ./cmd/juju/... -check.v
+        go test -v ./cmd/juju/... -check.v -coverprofile=coverage.txt -covermode=atomic
```

Comment:
Have we verified that running this both as a GH action and on jenkins for the same SHA does not clobber the coverage results?

Reply:
Not yet, but consider this is only an experiment, we can fix that if we move it to jenkins.